tools/regz/build.zig

```diff
@@ -14,12 +14,22 @@ pub fn build(b: *Build) !void {
         .iconv = false,
     });
 
+    const assets = [_]struct { []const u8, []const u8 }{
+        .{ "../../core/src/mmio.zig", "mmio_file" },
```

Comment:
You can't grab a file directly from another package like this. Instead you need to use the handle for the core package to retrieve it.

```suggestion
        .{ core_dep.path("src/mmio.zig"), "mmio_file" },
```

If this causes a dependency loop then MMIO needs to be its own package. I think this would be ideal, since we could put some tooling for simulated MMIO in there as well.

Reply:
@mattnite Yeah I see what you mean. if I use the core_dep to reference it then regz can no longer be built by running `zig build` in the tools/regz directory. Not sure if there is a way to tell in the build.zig if it is being run from a parent or in the current directory. Let me look into this a bit and see what I can do.

Reply:
I think it would be easier to just copy the `Mmio` struct in the regz project to be used in standalone mode, because you also need a definition of `Handler` and `unhandled`.

Reply:
@mattnite I tried to reference via core_dep but I can't seem to figure out how to make it work. None of the tools have any dependencies on the core module that I can use as a reference so I'm not quite sure how to access that dependency in the regz build.zig.  Sorry but I'm pretty new to zig and the build system on this project is a bit more complicated than I am used to. Can you provide some guidance on how to access within the build.zig of regz?